### DISCUSSION TITLE
Is Istio causing any network drop or connection issues between an application  pod and RDS ?

### DISCUSSION THREAD

**Discussion by Govind10g** [istio/istio#51240] [Q&A]
Hi Istio Team

We are using istio mesh in our application deployment which is running on AWS EKS cluster and our istio configuration includes istiod, istio-ingress-gateway, destination rule, peer-auth, Gateway, Istio-operator, ServiceEntry and EnvoyFilter.

And  with above configuration on Istio, we are able to access the application and it is working fine however we are facing intermittent network connection issue between our application pod and AWS RDS database.

So we would like to know whether istio causing any network drop or connection issues between an application  pod and RDS database instance ? Is there any such kind of possibilities in istio side to effect the connection flow ?

Error is application log - psycopg2.InterfaceError: connection already closed.
Error is RDS log - could not receive data from client: Connection reset by peer.

Any insights or assistance you can provide would be greatly appreciated :pray:

**Comment by parshva-shah-dsw** [comment]
@Govind10g   are these connections issues seen during Istiod restart?  we are also facing connection being re-established https://github.com/istio/istio/discussions/53858
